www/src/output/site.css

```diff
@@ -0,0 +1,577 @@
+:root {
+    --body-max-width: 900px;
+    --text-color: #121212;
+    --top-bar-bg: #222;
+    --top-bar-fg: #eee;
+    --top-bar-logo-hover: hsl(258, 73%, 70%);
+    --header-link-color: #17b9b0;
+    --header-link-hover: #222;
+    --link-color: hsl(258, 73%, 58%);
+    --h1-color: hsl(258, 73%, 70%);
+    --repl-prompt: #0f50f4;
+    --body-bg: #fff;
+    --code-bg: #e7e7e7;
+    --language-snippet-bg: #fcfcfc;
+    --language-snippet-border: #bbb;
+    --code-color: #303030;
+    --toc-background: #f3f3f3;
+    --toc-border: #ddd;
+    --toc-search-bg: #fcfcfc;
+    --toc-search-border: #bbb;
+  }
+  
+  html {
```

Comment:
Were these extra spaces before `html` added by accident?

Reply:
Looks like vscode changes the spacing when copy-pasting CSS...
I'm going to undo changes to the spacing to minimize the diff. We can format CSS the correct way in another PR.

Reply:
Sounds good.